packages/grid/x-data-grid-premium/package.json

```
@@ -44,8 +44,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.23.9",
-    "@mui/system": "^5.15.6",
-    "@mui/utils": "^5.15.6",
+    "@mui/system": "^5.15.7",
```

This is a stable dependency, would make more sense to me
```suggestion
    "@mui/system": "^5.15.0",
```

---

What is the use-case? and why not `"^5.0.0"` then?
Do you know whether it's possible to configure renovatebot to respect this new dependency philosophy?